RNTester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -436,7 +436,7 @@ class AccessibilityActionsExample extends React.Component {
         </RNTesterBlock>
 
         <RNTesterBlock title="View with multiple actions">
-          <View
+          <TouchableOpacity
```

Comment:
Since you have added a new example using TouchableOpacity, no need to change this existing example.

Reply:
Your right I did not change it back because it behaved the same way, but no harm in leaving it alone either. I will change it back.